etc/systemd/wsdd-discovery.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Web Services Dynamic Discovery service
+Documentation=man:wsdd(8)
+Requires=wsdd-discovery.socket
+RefuseManualStart=true
+
+[Service]
+Type=simple
+; The service is put into an empty runtime directory chroot,
+; i.e. the runtime directory which usually resides under /run
+ExecStart=/usr/bin/wsdd --shortlog --chroot=/run/wsdd --source-port=37020 --no-host --discovery
```

Comment:
Do we want to use the same flags from `/etc/default/wsdd` here?

Reply:
Might help in some minor use-cases, I think (including verbosity for debugging, e.g.), so why not. But I propose to use a different environment file.

I also propose to use a different run/chroot directory for the wsdd-discovery service in order to separate it from a regular host instance. Maybe even a different (dynamic) username might be good, although I'm not sure if dynamic users are inherently distinct (due to different namespace) even when sharing the same name.

Reply:
> although I'm not sure if dynamic users are inherently distinct (due to different namespace) even when sharing the same name.

They're using the same UID/GID here, so no

Reply:
How's this?
I've used the same env file but a different variable name

Reply:
`--source-port 37020` could also be set in the environment file instead of hard-coded inthe systemd unit

Reply:
It would be nice to make that configurable from the environmental file, but it is hardcoded in the firewall configuration anyway...